packages/snaps-execution-environments/src/common/endowments/commonEndowmentFactory.ts

```diff
@@ -0,0 +1,77 @@
+import crypto from './crypto';
+import interval from './interval';
+import math from './math';
+import network from './network';
+import textDecoder from './textDecoder';
+import textEncoder from './textEncoder';
+import timeout from './timeout';
+
+export type EndowmentFactory = {
+  names: readonly string[];
+  factory: () => unknown;
+};
+
+export type CommonEndowmentSpecification = {
+  endowment: unknown;
+  name: string;
+};
+
+// Array of common endowments
+const commonEndowments: CommonEndowmentSpecification[] = [
+  { endowment: AbortController, name: 'AbortController' },
+  { endowment: AbortSignal, name: 'AbortSignal' },
```

Comment:
I thought the idea behind postponing Abort endowments was that we postpone adding them as endowments and not add them but postpone making them safe to add. 

Is this a decision that was made or just a result of circumstances?

Reply:
`AbortController` and `AbortSignal` are both part of out default endowments which are exported and can be used for a long time.
I have no clear instructions on removing any of the default endowments, so I have to keep them all. 
Anyway, they'll get harden() at least in this iteration.
Here is the file with default endowments already exported or in use: https://github.com/MetaMask/snaps-monorepo/blob/main/packages/snaps-utils/src/default-endowments.ts

//cc @ritave

Reply:
I'll approve the PR but encourage finishing this conversation before merge.

Reply:
We decided to leave these endowments as they were before exported in the defaults, then we will make a decision how and if we're going to keep them. We can address that work in the next ticket/PR: https://github.com/MetaMask/snaps-monorepo/issues/1132

Resolving this conversation for now.